digdag-standards/src/main/resources/digdag/standards/py/runner.py

```diff
@@ -122,7 +122,7 @@ def digdag_inspect_arguments(callable_type, exclude_self, params):
         if key in params:
             args[key] = params[key]
         else:
-            if spec.defaults is None or len(spec.defaults) < idx:
+            if spec.defaults is None or idx < len(spec.args) - len(spec.defaults):
```

Comment:
Just for note. 
`spec.args` returns all arguments in a list, while `spec.defaults` returns a tuple of default values.

For example:

```py
In [2]: def ng(arg1, arg2, arg3=3):
   ...:     print(arg1, arg2, arg3)

In [4]: spec = inspect.getargspec(ng)

In [5]: spec.args
Out[5]: ['arg1', 'arg2', 'arg3']

In [6]: spec.defaults
Out[6]: (3,)

In [7]: spec
Out[7]: ArgSpec(args=['arg1', 'arg2', 'arg3'], varargs=None, keywords=None, defaults=(3,))
```

Default arguments should be in the end of arguments, so we need to check front arguments.